.gitignore

```diff
@@ -74,3 +74,8 @@ ENV/
 
 # Fabric Files
 fabfile/
+
+# Frontend Modules
+node_modules
+frontend/bower_components
+frontend/node_modules
```

Comment:
Ok Sir , I didn't knew that. I will fix this . 
So shall I add a commit on the branch from which I sent the PR and open a new PR ?